src/StimulusBundle/src/DependencyInjection/StimulusExtension.php

```diff
@@ -38,6 +38,12 @@ public function load(array $configs, ContainerBuilder $container): void
             ->replaceArgument(2, $config['controller_paths'])
             ->replaceArgument(3, $config['controllers_json']);
 
+        if (null === $config['use_optimized_attributes_rendering']) {
+            trigger_deprecation('symfony/stimulus-bundle', '2.20', 'The configuration option "use_optimized_attributes_rendering" is deprecated. It is now interpreted as "false", but it will be interpreted as "true" (and removed) in the next major version. Configure "use_optimized_attributes_rendering" to "true" to use the new and optimized rendering system for Stimulus Attributes.', __METHOD__);
```

Comment:
This deprecation message makes me think I need to *remove* the option to solve it as it says the option is deprecated.
What is deprecated is actually *not configuring it*. See https://github.com/symfony/symfony/blob/eb88eb3eb820208f1e8f9ba89b319811753a4d12/src/Symfony/Bundle/FrameworkBundle/DependencyInjection/Configuration.php#L88 for the wording of such deprecation in Symfony.

Reply:
Thanks for the example!